src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -215,6 +187,29 @@
       Properties="Configuration=$(Configuration);PublishDir=$(OutputPath)/FSharp" />
   </Target>
 
+  <Target Name="PublishConatinersSdk"
+            BeforeTargets="Build">
+    <ItemGroup>
+      <BuildFiles Include="$(RepoRoot)/src/Containers/packaging/build/Microsoft.NET.Build.Containers.props" />
+      <BuildFiles Include="$(RepoRoot)/src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets" />
+    </ItemGroup>
+    <Copy SourceFiles="@(BuildFiles)" DestinationFolder="$(OutputPath)/Containers/build" />
+
+    <MSBuild
+      Targets="Publish"
+      Projects="$(RepoRoot)/src/Containers/Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj"
+      Properties="Configuration=$(Configuration);PublishDir=$(OutputPath)/Containers/tasks/net472;TargetFramework=net472"
+      Condition="'$(DotNetBuildFromSource)' != 'true'" />
+    <MSBuild
+      Targets="Publish"
+      Projects="$(RepoRoot)/src/Containers/Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj"
+      Properties="Configuration=$(Configuration);PublishDir=$(OutputPath)/Containers/tasks/net7.0;TargetFramework=net7.0" />
```

Comment:
Is there a prop to get the `net7.0` from so this line doesn't have to be updated manually/annually?